cores/esp8266/cont_util.c

```diff
@@ -38,13 +38,13 @@ void ICACHE_RAM_ATTR cont_init(cont_t* cont) {
     }
 }
 
-int ICACHE_RAM_ATTR cont_check(cont_t* cont) {
+int cont_check(cont_t* cont) {
```

Comment:
I'm calling this from an ISR, so it needs to be in RAM. 
My usage is as a kind of canary for stack usage. I suppose that would no longer be necessary when the breakpoint approach is implemented, but until then, I'd prefer this be kept in RAM.